app/models/name/format.rb

```diff
@@ -1,14 +1,23 @@
+# frozen_string_literal: true
```

Comment:
There is only one substantive change in this file: I added `short_display_name`.
Everything else is simply reorganization: I had a hard time figuring out where to put that method, so I moved some things around.